kaldb/pom.xml

```diff
@@ -14,17 +14,17 @@
         <uberjar.name>kaldb</uberjar.name>
         <http.client.version>4.5.13</http.client.version>
         <http.core.version>4.4.15</http.core.version>
-        <protobuf.version>3.20.1</protobuf.version>
-        <grpc.version>1.46.0</grpc.version>
-        <armeria.version>1.16.0</armeria.version>
-        <micrometer.version>1.8.5</micrometer.version>
-        <kafka.version>2.5.1</kafka.version>
+        <protobuf.version>3.21.1</protobuf.version>
+        <grpc.version>1.47.0</grpc.version>
+        <armeria.version>1.17.0</armeria.version>
+        <micrometer.version>1.9.1</micrometer.version>
+        <kafka.version>3.2.0</kafka.version>
         <jackson.version>2.13.3</jackson.version>
-        <jackson.databind.version>2.10.5.1</jackson.databind.version>
+        <jackson.databind.version>2.12.7</jackson.databind.version>
```

Comment:
This gets us to a version without any known vulnerabilities - https://snyk.io/vuln/maven:com.fasterxml.jackson.core%3Ajackson-databind